PL2/PL2-parser/src/main/java/nl/tudelft/pl2016gr2/parser/controller/AnnotationReader.java

```diff
@@ -0,0 +1,211 @@
+package nl.tudelft.pl2016gr2.parser.controller;
+
+import nl.tudelft.pl2016gr2.model.Annotation;
+import org.apache.poi.openxml4j.exceptions.InvalidFormatException;
+import org.apache.poi.ss.usermodel.Cell;
+import org.apache.poi.ss.usermodel.Row;
+import org.apache.poi.ss.usermodel.Sheet;
+import org.apache.poi.ss.usermodel.Workbook;
+import org.apache.poi.ss.usermodel.WorkbookFactory;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+@SuppressWarnings("checkstyle:AbbreviationAsWordInName")
+public class AnnotationReader {
+
+  private static final Logger logger = Logger.getLogger(AnnotationReader.class.getName());
+
+  private final File file;
+
+  public AnnotationReader(File file) {
+    this.file = file;
+  }
+
+  @SuppressWarnings("checkstyle:MethodLength")
+  private Annotation readRow(Row row) {
```

Comment:
This is a long method. Add javadoc to describe what it does and why the annotation is needed.